### PR TITLE
Default series toggle

### PR DESCRIPTION
**Pull request by zahardev**
## Summary

Allow admins to change the default podcast directly from the single series edit page, replacing the need for the SSA plugin's "Change Default Podcast" action.

## Requirements

- Non-default series edit page should display a toggle labeled "Set as default podcast"
- Toggling on and saving should update the `ss_podcasting_default_series` option to the current series
- The current default series should display a text label ("This is the default podcast") instead of a toggle
- Requires `manage_podcast` capability

## Summary by CodeRabbit

* **New Features**
  * Added a "Default Podcast" toggle in series settings.

* **Style**
  * New styles for the default-podcast label and a custom toggle control.

* **Bug Fixes**
  * Image delete now clears all associated fields.
  * Membership sync moved to queued single-sync processing and avoids conflicting add/revoke operations for more reliable syncing.

* **Tests**
  * Added tests covering membership sync queuing, merging, reconciliation, and edge cases.

### CODE REVIEW

File: php/classes/integrations/woocommerce/class-wc-memberships-integrator.php [hunk 1]
```diff
@@ -240,52 +240,114 @@ protected function init_single_sync_subscriber() {
...
+	 *
+	 * @return void
+	 */
+	protected function sync_membership( $membership ) {
+		if ( 'active' === $membership->get_status() ) {
+			$this->prepare_single_sync( $membership->get_user_id(), $membership->get_plan_id(), null );
+		} else {
+			$this->prepare_single_sync( $membership->get_user_id(), null, $membership->get_plan_id() );
+		}
+	}
```
[the remark this answers is not in the document]
Comment (zahardev) [reply php/classes/integrations/woocommerce/class-wc-memberships-integrator.php:255]: Valid finding, but this file belongs to a different feature (#838) and is not in scope for this PR. Will address in the appropriate branch.

File: php/classes/integrations/woocommerce/class-wc-memberships-integrator.php [hunk 2]
```diff
@@ -240,52 +240,114 @@ protected function init_single_sync_subscriber() {
...
+		}
 
-					// Make sure user doesn't have other memberships that allow them to use this series
-					$user_membership_ids = $this->get_user_membership_ids( $user_id );
-					$allowed_series_ids  = $this->convert_membership_ids_into_series_ids( $user_membership_ids );
-					$revoke_series_ids   = array_diff( $revoke_series_ids, $allowed_series_ids );
+		if ( isset( $snapshot['attempts'] ) && (int) $snapshot['attempts'] >= 10 ) {
+			$this->logger->log( __METHOD__ . ': Giving up after 10 failed attempts.' );
+			return;
+		}
```
[the remark this answers is not in the document]
Comment (zahardev) [reply php/classes/integrations/woocommerce/class-wc-memberships-integrator.php:285]: Valid finding, but this file belongs to a different feature (#838) and is not in scope for this PR. Will address in the appropriate branch.
